docs/pages/release_notes.md

```diff
@@ -65,6 +69,7 @@ This is a {{ site.pmd.release_type }} release.
     *   [#1578](https://github.com/pmd/pmd/issues/1578): \[java] Private field is detected as public inside nested classes in interfaces
 *   java-bestpractices
     *   [#658](https://github.com/pmd/pmd/issues/658): \[java] OneDeclarationPerLine: False positive for loops
+    *   [#1405](https://github.com/pmd/pmd/issues/1405): \[java] New Rule: UseTryWithResources - Replace close and IOUtils.closeQuietly with try-with-resources
```

Comment:
not for this PR, but I'm wondering… should we stop listing all closed issues and limit ourselves to those that are actually bugs / false positives / false negatives?

Everything else is either a chore for ourselves, a rule change / new rule we are already addressing in a specific section, or an enhancement we usually include as new and noteworthy…

I feel this is getting too noisy / giving the false perspective we have plenty of issues rather than we are actively developing the product